integrations/visual_studio.rst

```diff
@@ -4,114 +4,79 @@
 |visual_logo| Visual Studio
 ===========================
 
-Conan can be integrated with **Visual Studio** in two different ways:
+Microsoft Visual Studio is one of the most used IDEs to develop C++ applications and Conan has
```

Comment:
may be put some note to distinguish `Visual Studio`, `Visual Studio Code` and `Visual Studio for Mac`, explaining that it's only for desktop one

Reply:
I think that the official name for this one in "Microsoft Visual Studio", the others are the ones that require specification... but anyway, do you think it is necessary? Is it not enough with the logo? I suppose that people reading this piece of docs are somehow familiarized with C++ and some content can also be used with `VSCode`

Reply:
up to you :)